src/assets/style/main.scss

```diff
@@ -0,0 +1,6 @@
+$color__green:#008000;
```

Comment:
For scss variables they should be written in lowecase and use hyphens rather than underscore eg.  `$color-green`.

Reply:
Thanks let me change that one